ops/e2e.Dockerfile

```diff
@@ -133,12 +142,10 @@ RUN wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/nul
  && rm -rf /var/lib/apt/lists/*
 
 # Retrieve Triton server source
-ARG TRITON_VERSION=2.11.0
+ARG TRITON_VERSION=main
```

Comment:
Is this correct?  Not sure on triton versioning.

-- nevermind, it's the branch. I might suggest renaming TRITON_VERSION to TRITON_BRANCH

Reply:
Yep! I like that much better. Clarifies the distinction between the release version and the branch name. I'll upgrade accordingly.